geom_mgr/CubedSphere/CubedSphereGeomSpec.F90

```diff
@@ -0,0 +1,118 @@
+#include "MAPL_ErrLog.h"
+
+module mapl3g_CubedSphereGeomSpec
+   use mapl3g_GeomSpec
+   use mapl3g_CubedSphereDecomposition
+   use esmf, only: ESMF_KIND_R8, ESMF_CubedSphereTransform_Args
+   implicit none
+   real(kind=ESMF_Kind_R8) :: undef_schmidt = 1d15
+   private
+
+   public :: CubedSphereGeomSpec
+   public :: make_CubedSphereGeomSpec
+
+   type, extends(GeomSpec) :: CubedSphereGeomSpec
+      private
+      integer :: im_world
+      type(ESMF_CubedSphereTransform_Args) :: schmidt_parameters
```

Comment:
I wonder if we should point out to the ESMF team that these actually have nothing to do with CS.   We just happen to want to use them for the CS.   I can imagine someone wanting to do the same with an Icosahedral mesh.   (LatLon would seem a bit wonky, but even then could be used to move the pole problem to a location of choice.)